conf/machine/imx6sxsabreauto.conf

```diff
@@ -12,8 +12,8 @@ require conf/machine/include/tune-cortexa9.inc
 KERNEL_DEVICETREE = "imx6sx-sabreauto.dtb"
 KERNEL_DEVICETREE_use-mainline-bsp = "imx6sx-sabreauto.dtb"
 
-PREFERRED_PROVIDER_u-boot = "u-boot-imx"
-PREFERRED_PROVIDER_virtual/bootloader = "u-boot-imx"
+PREFERRED_PROVIDER_u-boot = "u-boot-fslc"
+PREFERRED_PROVIDER_virtual/bootloader = "u-boot-fslc"
```

Comment:
Please prepare the PR against `master` and then later we can cherry-pick to `dunfell`.

Reply:
just did, I'm sorry. I thought you would need the PR for both branches